evm/src/cpu/kernel/asm/bignum/modmul.asm

```diff
@@ -130,33 +142,38 @@ modmul_return_4:
     // STEP 6:
     // Check that x + k * m = a * b.
 
-    // Walk through scratch_2 and scratch_3, checking that they are equal.
```

Comment:
Any reason why the comment is deleted?

Reply:
Not sure why it appears deleted, it's still there 🤔